### PR TITLE
[TW-1562] Create new overview for Billing section

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**
* Added a new overview to the Billing section
* Minor updates on related pages

### CODE REVIEW

**Review by jkonrath-postman — APPROVED** [review]
LGTM

File: src/pages/docs/billing/billing-overview.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+updated: 2024-01-18
+contextual_links:
+  - type: section
+    name: "Additional resources"
+  - type: link
+    name: "Postman Pricing"
+    url: "https://www.postman.com/pricing/"
+---
+
+Postman provides several billing features that enable you to manage your team's payment methods, change your team's plan, and track your team's resource usage. With these billing features, you can ensure your team has access to the Postman features they need to be successful.
```
Comment (nerminamiller-postman) [inline src/pages/docs/billing/billing-overview.md:12]:
```suggestion
Learn how to manage your team's payment methods, change your team's plan, and track your team's resource usage in Postman. With these billing features, you can select and measure your team’s access to the Postman features and make scaling decisions that best fit your needs.
```

File: src/pages/docs/billing/billing-overview.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+For help with any account or billing issues, go to Postman's [Support Center](https://www.postman.com/support/) and select **Submit a request**.
+
+> Check out [Postman's Help Center](https://support.postman.com/hc/en-us) for [Postman billing and payment FAQs](https://support.postman.com/hc/en-us/categories/115000609205-Billing-and-Payment).
+
+## Contents
```
Comment (nerminamiller-postman) [inline src/pages/docs/billing/billing-overview.md:18]: Consider omitting the TOC due to the page length and SEO.

File: src/pages/docs/billing/billing-overview.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## Manage your account, payment methods, and plan
+
+To get started, you can learn how to [purchase and configure a paid Postman plan](/docs/billing/buying/).
+
+You can use the Postman billing dashboard to [view and manage your team's billing](/docs/billing/billing/#billing-dashboard). This includes an overview of your Postman plan, past invoices, and billing-related activity. You can [manage your team's payment methods](/docs/billing/billing/#payments). You can also [make changes to your plan](/docs/billing/billing/#team-and-plan-changes), including adjusting your team size, purchasing add-ons, and upgrading your plan.
+
+## Track your resource usage
+
+Depending on your [Postman plan](https://www.postman.com/pricing/), you have a specified number of resources you can use each month. You can use the Resource Usage dashboard to [view the resources your team is using](/docs/billing/resource-usage/) and view your team's remaining limits for the month.
```
Comment (nerminamiller-postman) [inline src/pages/docs/billing/billing-overview.md:31]:
```suggestion
The number of resources you can use depends on your [Postman plan](https://www.postman.com/pricing/). Check the Resource Usage dashboard to [view the resources your team is using](/docs/billing/resource-usage/) to view your team's remaining limits for the month.
```

File: src/pages/docs/billing/billing-overview.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+## Contents
+
+* [Manage your account, payment methods, and plan](#manage-your-account-payment-methods-and-plan)
+* [Track your resource usage](#track-your-resource-usage)
+
+## Manage your account, payment methods, and plan
+
+To get started, you can learn how to [purchase and configure a paid Postman plan](/docs/billing/buying/).
+
+You can use the Postman billing dashboard to [view and manage your team's billing](/docs/billing/billing/#billing-dashboard). This includes an overview of your Postman plan, past invoices, and billing-related activity. You can [manage your team's payment methods](/docs/billing/billing/#payments). You can also [make changes to your plan](/docs/billing/billing/#team-and-plan-changes), including adjusting your team size, purchasing add-ons, and upgrading your plan.
```
Comment (nerminamiller-postman) [inline src/pages/docs/billing/billing-overview.md:27]:
```suggestion
The Postman billing dashboard enables you to [view and manage your team's billing](/docs/billing/billing/#billing-dashboard). In the dashboard, you will find an overview of your Postman plan, past invoices, and billing-related activity. You can [manage your team's payment methods](/docs/billing/billing/#payments). You can also [make changes to your plan](/docs/billing/billing/#team-and-plan-changes), such as adjust your team size, purchase add-ons, and upgrade your plan.
```

**Review by nerminamiller-postman — APPROVED** [review]
Nice work, Matthew! Left you a few comments to consider as you see fit.

File: src/pages/docs/billing/billing-overview.md (hunk 3 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/billing/billing-overview.md:31]: I made updated based on your feedback. Thanks!

File: src/pages/docs/billing/billing-overview.md (hunk 4 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/billing/billing-overview.md:27]: I made updated based on your feedback. Thanks!

**Pull request merged**